examples/PasswordExpiryWatcher/README.md

```diff
@@ -0,0 +1,96 @@
+## Password Expiry Watcher
+
+This tool allows you as a system administrator to download a CSV of all of your users with each user's password expiration date. This allows monitoring of users with upcoming password expirations and for you the administrator to prevent these users from abruptly being cut out of their mission critical systems.
+
+### To Install
+
+##### I do not have the Go toolchain installed
+> Most users will want to follow these instructions
+
+1. Go to the "Releases" tab of this github project
+	- https://github.com/TheJumpCloud/jcapi/releases
+2. We have provided `.zip` files for most operating systems
+	- Note that macOS users will want to use the `darwin` binaries
+3. Download the zip file of your choice 
+	- We always recommend downloading from the latest release if possible
+	- Most Windows users will want the `386` zip
+4. Extract the files from the zip
+	- Right click on the zip file and click `Extract All` on Windows
+	- Double click on the zip file on macOS/Linux
+5. `PasswordExpiryWatcher_myos_myarch` is the relevant binary
+	- Some may find this is an unwieldy name to type every time. Feel free to rename to your liking, in fact if this is a tool you plan on running frequently we encourage this for ease of use.
+
+> If you'd like to be able to call this binary from an arbitrary directory you can move it to `/usr/local/bin` on Linux and macOS
+>> On macOS the keyboard shortcut `⌘ + Shift + G` with an open Finder window will allow you to directly type in the directory you'd like to access. To move the binary to `/usr/local/bin` with Finder simply use that shortcut and copy/paste `/usr/local/bin` into the dialog box. You can now drag and drop the binary onto the Finder window to easily make it accessible without having to navigate to a specific folder to run it. If you do this we also recommend renaming the binary before moving it to a more memorable name.
+
+##### I have the Go toolchain installed
+> If you don't know what "Go" or "Golang" is we have provided pre-made binaries for your convenience. Installation instructions for these binaries is provided in the previous section
+
+1. Clone the `jcapi` repository (it doesn't matter where)
+	- `git clone https://github.com/TheJumpCloud/jcapi`
+2. Navigate to the `jcapi/examples/PasswordExpiryWatcher` directory
+	- `cd jcapi/examples/PasswordExpiryWatcher`
+3. Run go install
+	- `go install .`
+
+This will install a binary called `passwordExpiryWatcher` to your `$GOBIN`
+
+> If you'd like to be able to call this binary from an arbitrary directory make sure your `$GOBIN` is in your `$PATH` (linux) or `%PATH` (windows)
+
+### To Run
+
+To run this tool you will need to use what is called the "command line". On Windows this is a program called "Power Shell", on macOS it is an app called "Terminal". 
```

Comment:
this is kind of condescending. I'd just say something like `To run this tool you will need to use the command line. On Windows you can use PowerShell and on OSX you can use Terminal`. Also, PowerShell is one word!